hadoop-ozone/common/pom.xml

```diff
@@ -107,6 +107,10 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <groupId>org.junit.jupiter</groupId>
       <artifactId>junit-jupiter-params</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.ozone</groupId>
+      <artifactId>hdds-server-framework</artifactId>
+    </dependency>
```

Comment:
`ozone-common` is shared by Ozone client and server code, thus it should not depend on `hdds-server-framework` (server-side stuff).

Reply:
done

Reply:
It's still there:

https://github.com/apache/ozone/blob/dbc31173a2982ea8d96a66c64aab9d21f4b4104b/hadoop-ozone/common/pom.xml#L110-L113

Reply:
removed it